tracing-core/src/metadata.rs

```diff
@@ -283,6 +283,19 @@ impl Level {
     ///
     /// Designates very low priority, often extremely verbose, information.
     pub const TRACE: Level = Level(LevelInner::Trace);
+
+    /// Get the string representation of the `Level`.
```

Comment:
this is admittedly quite nitpicky, but in general, i think "get" should be preferred only for functions that do some kind of lookup or data structure access. I'd say
```suggestion
    /// Returns the string representation of the `Level`.
```